### ISSUE TITLE
Can't create git repo in macOS

### ISSUE THREAD

**Issue by JonaDupIO** [keybase/client#8841]
Hi

Error in CLI and API.
Seems that I can't chat anymore with people I added before I reinstall my Mac.
I tried Keybase long time ago, delete it, and then I try to use it to test git and teams features.

My log id: 270c0b1cffb26c81d2bdaa1c

Thanks!

**Comment by maxtaco** [comment]
Can you send a screenshot?

**Comment by JonaDupIO** [comment]
<img width="682" alt="capture d ecran 2017-10-05 a 16 56 13" src="https://user-images.githubusercontent.com/7522290/31234120-5e98a56e-a9ee-11e7-966c-9a23517d00d6.png">

**Comment by maxtaco** [comment]
yeah, it's because you reset your account and have all new keys, it's cryptographically impossible for you to gain access to your old data until you are rekeyed by your collaborators. We warn pretty heavily about this on reset.

**Comment by JonaDupIO** [comment]
I understand about the chat/files part. 
But why I have the same problem with git?

**Comment by maxtaco** [comment]
Git is build on top of the file system. Sorry for any confusion there

**Comment by JonaDupIO** [comment]
Ok, thanks.
What can I do to fix this?
I need a new account?

**Comment by maxtaco** [comment]
No, i think the people you're working with need to come back online and rekey for you.  @strib do you know the specifics of how a git repo should work with a reset user?

**Comment by strib** [comment]
@JonaDupIO: from your logs, it looks like you're trying to access a team repo, not a multi-user repo (like your screenshot above implies).  That should work, assuming the repo has been created via the GUI or command line, and your current, post-reset account is a member of that team.

What output are you getting on the git command line?

**Comment by JonaDupIO** [comment]
`➜  test keybase git create test`
`libkb.NeedSelfRekeyError{Tlf:"", Msg:"This device does not yet have read access to directory /keybase/private/jonathandupuich, log into Keybase from one of your other devices to grant access: --- is not readable by jonathandupuich (uid:---)\ngithub.com/keybase/kbfs/libkbfs.isReadableOrError\n\t/Users/build/go/src/github.com/keybase/kbfs/libkbfs/md_util.go:74\ngithub.com/keybase/kbfs/libkbfs.(*KBFSOpsStandard).getMaybeCreateRootNode\n\t/Users/build/go/src/github.com/keybase/kbfs/libkbfs/kbfs_ops.go:480\ngithub.com/keybase/kbfs/libkbfs.(*KBFSOpsStandard).GetOrCreateRootNode\n\t/Users/build/go/src/github.com/keybase/kbfs/libkbfs/kbfs_ops.go:518\ngithub.com/keybase/kbfs/libgit.getOrCreateRepoAndID\n\t/Users/build/go/src/github.com/keybase/kbfs/libgit/repo.go:214\ngithub.com/keybase/kbfs/libgit.CreateRepoAndID\n\t/Users/build/go/src/github.com/keybase/kbfs/libgit/repo.go:328\ngithub.com/keybase/kbfs/libgit.(*RPCHandler).CreateRepo\n\t/Users/build/go/src/github.com/keybase/kbfs/libgit/rpc.go:172\ngithub.com/keybase/kbfs/vendor/github.com/keybase/client/go/protocol/keybase1.KBFSGitProtocol.func2\n\t/Users/build/go/src/github.com/keybase/kbfs/vendor/github.com/keybase/client/go/protocol/keybase1/kbfs_git.go:58\ngithub.com/keybase/kbfs/vendor/github.com/keybase/go-framed-msgpack-rpc/rpc.(*callRequest).Serve\n\t/Users/build/go/src/github.com/keybase/kbfs/vendor/github.com/keybase/go-framed-msgpack-rpc/rpc/request.go:76\ngithub.com/keybase/kbfs/vendor/github.com/keybase/go-framed-msgpack-rpc/rpc.(*receiveHandler).handleReceiveDispatch.func1\n\t/Users/build/go/src/github.com/keybase/kbfs/vendor/github.com/keybase/go-framed-msgpack-rpc/rpc/receiver.go:121\nruntime.goexit\n\t/usr/local/Cellar/go/1.9/libexec/src/runtime/asm_amd64.s:2337"}`

**Comment by strib** [comment]
Ohh, I see, this is just your personal private folder, I was confused, sorry.  I guess you reset your account before we had auto-folder-resetting.

If you want us to reset your folder, please run this command on an currently-valid keybase device, and substitute the current date and time where indicated:

```
keybase sign -m "<DATE_AND_TIME>: Please reset folder /keybase/private/jonathandupuich because all the devices that have access have been revoked."
```

and post the results here.

**Comment by JonaDupIO** [comment]
BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkKNwdzf kcTHO5jd0qLyPC2 g6qzhri8kzFi6wZ 5xZh4ZXRXGrLFou vsjGfynTJ6fQiYj INvMe7EpJGuL5DP qv23Puy6Nm5PSnY SKc9baeAybVdEHh NrNG1cPcS5xn7qE yesQaR5kEmiNiN8 tEJx0BWjc5Eqi38 6qc0YhaFjaJ9Da4 dLqrbxZ4zH2QMOb Iygf47f5LThCulN 7tpmOVAMX9a27TL ZhHGSVH7FfEGN0o 6qoP0WFLYdzPDZl spfLRDRhGe89vmN r8DkdwE2uQmDkFH oPdgswPyquuTH19 XCOdFa3P110wTzF 3tOqauDe2nWZtwC FMilwOy5VjrLDbZ T0Y702A. END KEYBASE SALTPACK SIGNED MESSAGE.


Thanks a lot!

**Comment by strib** [comment]
I reset the folder.  Please check it out and close this if everything seems ok.  (Might require a Keybase restart.)

**Comment by maksim77** [comment]
Hi @strib!
I have the same problem! Please reset my folder:

> BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkIj8oen zlvptRsys9Rlh4o YLGptVFKC9qrxq0 l0iPiQWO1QWYbUu Q2UgJizsd4W9rdA 3Up8yWZSMot420z KBHXguASs0IdXLJ prEsonY9QJe6Gyb FcdnAZRiyFaang5 iMMuhCWueoRLu0n GAnaRCzdVNIirGV V3XlKamG3oABxQn TN2O12W2uKkJdnZ RVvLEJ4OQO97AIo XwSyMjs05DruYA7 4w8Jw8OlGxs29KU pj7S4RdVknTaSDO IeZbAYPUjlHzMb7 I9ZGwo4kZEF5LFC qNo2Lvhn5JaNDLB wX7MWE0Po1DGxzH i6aSrK3SAItsrF9 t1GFjoZG7lhFRm. END KEYBASE SALTPACK SIGNED MESSAGE.

**Comment by strib** [comment]
@maksim77: I reset that folder too.

**Comment by samkhawse** [comment]
@strib 
I've the same problem with my private folder. Could you please reset my folder?

> BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkTKTw0I IX1um4NXneemjTh F0NZGRDQ2OyNoCr eQ5QTyI6MqpbCrm qsu6D7LmGkb1FfV Esn4gsANCn2NO04 GlovQ1cW4yFIWKL 5YtpcpVDIbMx8kG HJXNNcEANTeetP1 VGVMAfG8XNnL0Lm DmLQ7oa00WPV5Ty GHfBCLlGLTxg9Vv uYWPYmDhwO5A7FQ Wjx37Jl6bBLgTLH AO1D5zaVdlKqNpL BbgP4TbubSJfepe oKLSmbWfBwEksAP 1mMwCIjQgSrHX3t yG4bEq0mccytZW2 H7g3sRaBJPKPWaN G2wwVjg1xA7pZXA 78yg95zY5VoUoVG QSLE1Z0paOpCaO0 E01sxRGe1XC. END KEYBASE SALTPACK SIGNED MESSAGE.

**Comment by strib** [comment]
@samkhawse: done.

**Comment by rmi7** [comment]
I'm having the same problem after I reinstalled my mac. I'm trying to create a new **personal** git repo. Any help is much appreciated!
